vtkWrapClientServer fails on vtkOMETiffReader.h on macOS

The error message is

```
vtkWrapClientServer: In /Users/cory.quammen/bin/tomviz-superbuild/tomviz/src/tomviz/tomviz/pvextensions/vtkOMETiffReader.h:30: syntax error.
make[2]: *** [tomviz/pvextensions/vtkOMETiffReaderClientServer.cxx] Error 1
make[1]: *** [tomviz/pvextensions/CMakeFiles/tomvizExtensions.dir/all] Error 2
```

The code at vtkOMETiffReader.h30 is

```
public:                                                                                                                                                                                                                                    
  static vtkOMETiffReader *New();                                                                                                                                                                                                          
  vtkTypeMacro(vtkOMETiffReader, vtkImageReader2)                                                                                                                                                                                          
  void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;  
```

Other classes follow this pattern and `vtkWrapClientServer` does not have a problem with them, so I'm not sure what the cause is. Adding a semicolon at the end of the line

```
vtkTypeMacro(vtkOMETiffReader, vtkImageReader2)    
```

fixes the problem for `vtkWrapClientServer`, but leads to compilation warning about an extra semicolon. I don't see what is wrong with the existing code.

I am using an older XCode, version 7.1.1, with Apple LLVM version 7.0.0 (clang-700.1.76).

--- a/tomviz/pvextensions/vtkOMETiffReader.h
+++ b/tomviz/pvextensions/vtkOMETiffReader.h
@@ -26,7 +26,7 @@ class vtkOMETiffReader : public vtkImageReader2
 {
 public:
   static vtkOMETiffReader *New();
-  vtkTypeMacro(vtkOMETiffReader, vtkImageReader2)
+  vtkTypeMacro(vtkOMETiffReader, vtkImageReader2);
   void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
 
   /**

